src/function/scalar/string/length.cpp

```diff
@@ -2,32 +2,13 @@
 
 #include "duckdb/common/exception.hpp"
 #include "duckdb/common/vector_operations/vector_operations.hpp"
+#include "util.hpp"
 #include "utf8proc.hpp"
 
 using namespace std;
 
 namespace duckdb {
 
-// length returns the size in characters
-struct StringLengthOperator {
```

Comment:
Same for this one as for the SubstringFun: please add this as a static method to `struct LengthFun`: `static int64_t Length(string_t input);` and call it as follows: `LengthFun::Length(input)` where it is needed.

Reply:
Done, PTAL